examples/digital_fingerprinting/production/conda_env.yml

```diff
@@ -29,5 +29,7 @@ dependencies:
     - librdkafka
     - mlflow>=2.2.1,<3
     - nodejs=17.4.0
+    - nvtabular=23.06
     - papermill
+    - requests=2.26.0
```

Comment:
No longer necessary after https://github.com/nv-morpheus/Morpheus/pull/1053 is merged